Sources/Misc/Deprecations.swift

```diff
@@ -146,7 +146,7 @@ public extension Purchases {
     /**
      * Enable automatic collection of Apple Search Ads attribution. Defaults to `false`.
      */
-    @available(*, deprecated, message: "Use Purchases.attribution.enableAdServicesAttributionTokenCollection() instead")
+    @available(*, deprecated, renamed: "attribution.enableAdServicesAttributionTokenCollection")
```

Comment:
it's not really a rename since it's now a method, is that ok?

Reply:
Oh shoot that's right. Then `message` is better 👍🏻